localstack/services/ses/ses_starter.py

```diff
@@ -81,6 +81,23 @@ def list_templates(self):
 
     email_responses.list_templates = list_templates
 
+    def delete_template(self):
+        template_name = self._get_param('TemplateName')
+        templates = ses_backend.templates
+        if template_name in templates:
+            del templates[template_name]
+        ses_backend.templates = templates
+        DELETE_IDENTITY_RESPONSE = """<DeleteTemplateResponse xmlns="http://ses.amazonaws.com/doc/2010-12-01/">
```

Comment:
nitpick: We usually use UPPERCASE variable names only for global variables at the root scope (shouldn't hold back the merge, though - can be tackled in a future PR).